src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -767,49 +800,30 @@ public function theProductSInventoryHasBecameTrackedWithItems(ProductInterface $
         $this->objectManager->flush();
     }
 
-    /**
-     * @param string $transition
-     */
-    private function applyShipmentTransitionOnOrder(OrderInterface $order, $transition)
+    private function applyShipmentTransitionOnOrder(OrderInterface $order, string $transition): void
     {
         foreach ($order->getShipments() as $shipment) {
             $this->stateMachineFactory->get($shipment, ShipmentTransitions::GRAPH)->apply($transition);
         }
     }
 
-    /**
-     * @param string $transition
-     */
-    private function applyPaymentTransitionOnOrder(OrderInterface $order, $transition)
+    private function applyPaymentTransitionOnOrder(OrderInterface $order, string $transition): void
```

Comment:
`/** @throws SMException */`

Reply:
I've referred to this case in the other comment